examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+    "outputId": "67002074-b771-4653-f9ff-8d60840fa972"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch: 5/7 Step: 1500/1500 100.00% |████████████████████|22.13s loss: 0.019525 val_loss: 0.019379\n",
+      "Epoch 5: val_loss improved from 0.01980 to 0.01938, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 6/7 Step: 1500/1500 100.00% |████████████████████|21.99s loss: 0.019125 val_loss: 0.019044\n",
+      "Epoch 6: val_loss improved from 0.01938 to 0.01904, saving file to saves/best_weight.ckpt\n",
+      "Epoch: 7/7 Step: 1500/1500 100.00% |████████████████████|21.24s loss: 0.018845 val_loss: 0.018818\n",
+      "Epoch 7: val_loss improved from 0.01904 to 0.01882, saving file to saves/best_weight.ckpt\n",
+      "Restoring model from saves/best_weight.ckpt\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[{'epoch': 5,\n",
+       "  'loss': 0.019525130399813254,\n",
+       "  'time': 22.13099593600009,\n",
+       "  'val_loss': 0.01937860634426276},\n",
+       " {'epoch': 6,\n",
+       "  'loss': 0.01912545581596593,\n",
+       "  'time': 21.99487811600011,\n",
+       "  'val_loss': 0.019044068900247416},\n",
+       " {'epoch': 7,\n",
+       "  'loss': 0.018844680001338324,\n",
+       "  'time': 21.23843467100005,\n",
+       "  'val_loss': 0.018818224750459193}]"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {
+      "tags": []
+     },
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model.fit_generator(trn_dataloader, val_dataloader, epochs=7, callbacks=callbacks, initial_epoch=num_epochs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "sxcPMY6Nw_P5"
+   },
+   "source": [
+    "# Reconstructed images after the second training process"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 332
+    },
+    "id": "453gF7TEw_P6",
+    "outputId": "65e868e0-6fba-484b-c68f-45c6b6867904"
+   },
+   "outputs": [
+    {
+     "data": {
+      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAlYAAAE7CAYAAAAfJ88GAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAAgAElEQVR4nOy92W9d15U3+Dt3nueBl5eDSIqibNGK7Hgox/GQxHFqTFB5+FD1AY1q1Afk6XtooB866L+gnhro1wKqge6ggOpUdReqglSlk1TsxHHkSJZdlmSJGkiKMy/vPI+8px+Utbzv4ZnuJSnRVXcBAimeffawztprr3lLsixjDGMYwxjGMIYxjGEMxwfL057AGMYwhjGMYQxjGMN/FBgLVmMYwxjGMIYxjGEMJwRjwWoMYxjDGMYwhjGM4YRgLFiNYQxjGMMYxjCGMZwQjAWrMYxhDGMYwxjGMIYTgrFgNYYxjGEMYxjDGMZwQnBqgpUkSb8vSdI9SZIeSpL0/dMaZwxjGMMYxjCGMYzhrIB0GnWsJEmyArgP4JsAtgFcB/DnsizfOfHBxjCGMYxhDGMYwxjOCNhOqd+XATyUZXkNACRJ+jsA3wGgKlhJkjSuUjqGMYxhDGMYwxi+KJCTZTmu9uC0XIFpAFvC/7d/97cxjGEMYxjDGMYwhi86bGg9OC2LlSFIkvQ9AN97WuOPYQxjGMMYxjCGMZw0nJZgtQNgWvj/1O/+xiDL8l8D+Gtg7AocwxjGMIYxjGEM/zHgtASr6wAWJUmaw2OB6s8A/NdhO
```

Comment:
```suggestion
    "One of the strong and usefull tools in Poutyne is the evaluate method, which not only provides you with the evaluation metrics, but also provides the ground truths and the predictions, if related arguments has been set True (as below)."
```

Reply:
has been set to `True` ?

(adding to)
(adding highlight of true).